config.json

```diff
@@ -71,10 +77,24 @@
         "slug": "accumulate",
         "name": "Accumulate",
         "uuid": "b2e12f15-5cae-4848-9ee8-2e1b0992b69c",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "lambdas",
+          "foreach",
+          "yield"
+        ],
+        "prerequisites": [
+          "lambdas",
+          "generic-types",
+          "foreach",
+          "enumerables",
+          "extension-methods"
+        ],
         "difficulty": 1,
-        "topics": []
+        "topics": [
+          "extension_methods",
+          "sequences",
+          "transforming"
+        ]
```

Comment:
Once an exercise has practices and prerequisites, you can remove the topics key.